build-scripts/docker/repo-test/generate-repo-files.py

```diff
@@ -206,20 +206,26 @@ def run (tag, arch='x86_64'):
         cp = subprocess.run(['docker', 'run', '--platform', 'linux/arm64/v8',
                              tag],
                             stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-    version = re.search(r'(PowerDNS Authoritative Server|PowerDNS Recursor|' +
+    _version = re.search(r'(PowerDNS Authoritative Server|PowerDNS Recursor|' +
                         r'dnsdist) (\d+\.\d+\.\d+(-\w+)?[^ ]*)',
                         cp.stdout.decode())
+    _features = re.search(r'[Ff]eatures: (.*)', cp.stdout.decode())
+
+    version = features = None
+    if _version:
+        version = _version.group(2)
+
+    if _features:
+        features = _features.group(1)
+
     if g_verbose:
         print(cp.stdout.decode())
     # for some reason 99 is returned on  `cmd --version` :shrug:
     if cp.returncode != 0 and cp.returncode != 99:
         # FIXME write failed output to log
         print('Error running {}: {}'.format(tag, repr(cp.returncode)))
-        return cp.returncode, None
-    if version and version.group(2):
```

Comment:
in case you're wondering about the removal of the `and ..` bit of this `if`: it never did anything. It would either be true (because the expression matched two groups) or throw an exception (but that never happens because then `version` would have been `None` already)